libraries/cmake/source/librdkafka/CMakeLists.txt

```diff
@@ -173,12 +176,19 @@ function(generateLibrdkafkaxx)
     target_compile_definitions(thirdparty_librdkafka_cxx PUBLIC
     LIBRDKAFKA_STATICLIB
     )
+    configure_file(
+	"${library_root}/../../../zstd/src/lib/common/zstd_errors.h"
+	"${CMAKE_CURRENT_BINARY_DIR}/include/librdkafka/zstd_errors.h"
```

Comment:
Please don't do this from librdkafka.
Each library should be responsible to public its own headers; do this from libzstd.

Reply:
yup just testing atm

Reply:
just something thats driving me nuts:
target_include_directories (thirdparty_librdkafka_c pri

> Beware that you have probably mixed a merge with a rebase and now the PR it's including commits that were already merged in  i was afraid of that " visual studio conflict resolution " on the rebase ..

Reply:
git log updated